README.md

```diff
@@ -18,6 +20,17 @@ Standard scientific python toolkit.
 ## How?
 We've used a stripped down version of 3D rendering provided by the [Allen Cell Explorer's](https://allencell.org) [3D Cell Viewer.](https://github.com/dmt-aics/volume-viewer)
 
+
+
+### Demo Notebooks
+
+- [Foam 3D](https://mybinder.org/v2/gh/kmader/ivvv/patch-2?filepath=examples%2Ffoam_3d.ipynb)
```

Comment:
Same as above - in order to merge this into imaging-tools:master shouldn't all the gh/kmader lines be changed to gh/imaging-tools ?

Reply:
Yes I didn't want to change them because the links then don't work (until the PR is accepted) but I can change them now